lib/pwnlib/tubes/tube.rb

```diff
@@ -236,6 +281,8 @@ def interact
             $stdout.write(s)
           end
         end
+      rescue
```

Comment:
Should only rescue `EOFError`.

Reply:
Future work: Use our own Exception class